packages/devkit/src/utils/convert-nx-executor.ts

```diff
@@ -36,7 +28,8 @@ export function convertNxExecutor(executor: Executor) {
         projectsConfigurations,
         nxJsonConfiguration,
         cwd: process.cwd(),
-        projectGraph,
+        projectGraph: null,
```

Comment:
Why are we not passing in the project graph anymore?

Reply:
Agreed. Overall, the docs around Nx and Angular need a good review and more details about the integration and caveats. I have that on my list of things to do. This is not the only thing that's not documented properly.